argo-cd-apps/overlays/development/kustomization.yaml

```diff
@@ -151,3 +152,8 @@ patches:
       kind: ApplicationSet
       version: v1alpha1
       name: spacerequest-cleaner
+  - path: development-overlay-patch.yaml
```

Comment:
This will deploy it on the openshift-ci cluster spawned for the e2e tests, do we want this?

Reply:
I added a comment about the cluster role to help clarify.